tts/Object.def.lua

```diff
@@ -494,3 +491,13 @@ end
 ---@return tts__Object[]
 function getAllObjects()
 end
+
+--- Render vector lines from a list of parameters, relative the aassoicated objects
+---@param parameters tts__VectorLineGroup[]
+function Object.setVectorLines(parameters) end
+
+--- Returns a table of sub-tables, each sub-table representing VectorLineGroup.
```

Comment:
What about?

> Returns an array of vector line groups?

I know that the official TTS docs is devoid of the word "array", however seems as these are types for Luanalysis, which has a concept of arrays, I think we can take some creative liberty 😛